### PR TITLE
Fix typos, improve, update at main-page files and .github files

### PR DESCRIPTION
**Pull request by standardAI**
# What does this PR do?

Hi! Thank you for this awesome library!
I am new to open source. Firstly, I am reading the documentation. Secondly, I will read the codebase. Thirdly, I plan to make documentation and code contributions. I am open to your guidance.
I would be grateful for your feedback. @stevhliu @yiyixuxu @patrickvonplaten

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [X] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by DN6 — APPROVED** [review]
LGTM 👍🏽

**Comment by DN6** [comment]
cc'ing: @stevhliu as well.

File: .github/ISSUE_TEMPLATE/translate.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (stevhliu) [inline .github/ISSUE_TEMPLATE/translate.md:1]: Nice addition!

File: .github/ISSUE_TEMPLATE/translate.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/diffusers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.
+
+Some notes:
+
+* Please translate using an informal tone (imagine you are talking with a friend about Diffusers 🤗).
+* Please translate in a gender-neutral way.
+* Add your translations to the folder called `<languageCode>` inside the [source folder](https://github.com/huggingface/diffusers/tree/main/docs/source).
+* Register your translation in `<languageCode>/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/diffusers/blob/main/docs/source/en/_toctree.yml).
+* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @patrickvonplaten for review.
```
Comment (stevhliu) [inline .github/ISSUE_TEMPLATE/translate.md:26]:
```suggestion
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu for review.
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ We as members, contributors, and leaders pledge to make participation in our
 community a harassment-free experience for everyone, regardless of age, body
 size, visible or invisible disability, ethnicity, sex characteristics, gender
 identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
+nationality, personal appearance, race, caste, color, religion, or sexual identity
```
Comment (stevhliu) [inline CODE_OF_CONDUCT.md:10]: No need to change anything here, these two are already covered under the existing code of conduct

File: CITATION.cff [hunk 4]
```diff
@@ -31,10 +31,11 @@ keywords:
...
   - image-generation
+  - hacktoberfest
   - diffusion
   - text2image
   - image2image
   - score-based-generative-modeling
   - stable-diffusion
+  - stable-diffusion-diffusers
 license: Apache-2.0
-version: 0.12.1
```
Comment (stevhliu) [inline CITATION.cff:40]: Let's keep the version (maybe it can be updated)

File: PHILOSOPHY.md [hunk 5]
```diff
@@ -22,22 +22,22 @@ In a nutshell, Diffusers is built to be a natural extension of PyTorch. Therefor
...
-- Diffusers aim at being a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx`, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
+- Diffusers aims at being a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx`, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
```
Comment (stevhliu) [inline PHILOSOPHY.md:25]:
```suggestion
- Diffusers aims to be a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx`, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
```

File: PHILOSOPHY.md [hunk 6]
```diff
@@ -22,22 +22,22 @@ In a nutshell, Diffusers is built to be a natural extension of PyTorch. Therefor
...
 - Diffusers prefers simple, self-explainable code over condensed, magic code. This means that short-hand code syntaxes such as lambda functions, and advanced PyTorch operators are often not desired.
 
 ## Simple over easy
 
-As PyTorch states, **explicit is better than implicit** and **simple is better than complex**. This design philosophy is reflected in multiple parts of the library:
+As The Zen of Python states, **explicit is better than implicit** and **simple is better than complex**. This design philosophy is reflected in multiple parts of the library:
```
Comment (stevhliu) [inline PHILOSOPHY.md:30]: This philosophy is actually inspired by PyTorch so lets keep it

File: PHILOSOPHY.md [hunk 7]
```diff
@@ -83,14 +83,14 @@ The following design principles are followed:
...
 - Models **do not** follow the single-file policy and should make use of smaller model building blocks, such as [`attention.py`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention.py), [`resnet.py`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/resnet.py), [`embeddings.py`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/embeddings.py), etc... **Note**: This is in stark contrast to Transformers' modeling files and shows that models do not really follow the single-file policy.
-- Models intend to expose complexity, just like PyTorch's module does, and give clear error messages.
+- Models intend to expose complexity, just like PyTorch's Module does, and give clear error messages.
```
Comment (stevhliu) [inline PHILOSOPHY.md:86]: No need to capitalize "module"

File: PHILOSOPHY.md [hunk 8]
```diff
@@ -83,14 +83,14 @@ The following design principles are followed:
...
 - The model design is a difficult trade-off between keeping code readable and concise and supporting many model checkpoints. For most parts of the modeling code, classes shall be adapted for new model checkpoints, while there are some exceptions where it is preferred to add new classes to make sure the code is kept concise and
-readable longterm, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
+readable long term, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
```
Comment (stevhliu) [inline PHILOSOPHY.md:93]:
```suggestion
readable long-term, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
```

File: README.md [hunk 9]
```diff
@@ -1,3 +1,19 @@
+<!---
+Copyright 2023- The HuggingFace Team. All rights reserved.
```
Comment (stevhliu) [inline README.md:2]:
```suggestion
Copyright 2022 - The HuggingFace Team. All rights reserved.
```

File: README.md [hunk 10]
```diff
@@ -202,9 +218,9 @@ just hang out ☕.
 - https://github.com/deep-floyd/IF
 - https://github.com/bentoml/BentoML
 - https://github.com/bmaltais/kohya_ss
-- +3000 other amazing GitHub repositories 💪
+- +6000 other amazing GitHub repositories 💪
```
Comment (stevhliu) [inline README.md:221]: Is this number accurate @patrickvonplaten @sayakpaul ?

**Review by stevhliu — COMMENTED** [review]
Nice, thanks for reviewing even the setup files for typos 🙂

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (standardAI) [reply CODE_OF_CONDUCT.md:10]: These two words were added in the 2.1 version:
https://www.contributor-covenant.org/version/2/1/code_of_conduct/

Comment (stevhliu) [reply CODE_OF_CONDUCT.md:10]: Ah ok, nevermind then! 👍

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -28,19 +28,19 @@ the core library.
...
 * 7. Contribute to the [examples](https://github.com/huggingface/diffusers/tree/main/examples).
 * 8. Fix a more difficult issue, marked by the "Good second issue" label, see [here](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+second+issue%22).
 * 9. Add a new pipeline, model, or scheduler, see ["New Pipeline/Model"](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+pipeline%2Fmodel%22) and ["New scheduler"](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+scheduler%22) issues. For this contribution, please have a look at [Design Philosophy](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md).
 
 As said before, **all contributions are valuable to the community**.
 In the following, we will explain each contribution a bit more in detail.
 
-For all contributions 4.-9. you will need to open a PR. It is explained in detail how to do so in [Opening a pull request](#how-to-open-a-pr)
+For all contributions 4.-9. you will need to open a PR. It is explained in detail how to do so in [Opening a pull request](#how-to-open-a-pr).
```
Comment (stevhliu) [inline CONTRIBUTING.md:43]:
```suggestion
For all contributions 4-9, you will need to open a PR. It is explained in detail how to do so in [Opening a pull request](#how-to-open-a-pr).
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -156,21 +156,21 @@ You can open a request for a model/pipeline/scheduler [here](https://github.com/
...
-more precise, provide the link to a duplicated issue or redirect them to [the forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63) or [Discord](https://discord.gg/G7tWnz98XR)
+more precise, provide the link to a duplicated issue or redirect them to [the forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63) or [Discord](https://discord.gg/G7tWnz98XR).
 
 If you have verified that the issued bug report is correct and requires a correction in the source code,
 please have a look at the next sections.
 
 For all of the following contributions, you will need to open a PR. It is explained in detail how to do so in the [Opening a pull request](#how-to-open-a-pr) section.
 
-### 4. Fixing a "Good first issue"
+### 4. Fixing a `Good first issue`
```
Comment (stevhliu) [inline CONTRIBUTING.md:173]: Better to keep it as it is because the code formatting isn't rendered in the right navbar

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -271,17 +271,17 @@ with Diffusers.
...
 - A link to some training results (logs, models, ...) that show what the user can expect as shown [here e.g.](https://api.wandb.ai/report/patrickvonplaten/xm6cd5q5).
-- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
+- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
 
 If you are contributing to the official training examples, please also make sure to add a test to [examples/test_examples.py](https://github.com/huggingface/diffusers/blob/main/examples/test_examples.py). This is not necessary for non-official training examples.
 
-### 8. Fixing a "Good second issue"
+### 8. Fixing a `Good second issue`
```
Comment (stevhliu) [inline CONTRIBUTING.md:278]: Also keep the quotes here instead of the ticks

**Comment by patrickvonplaten** [comment]
Feel free to merge once you're happy with the PR @stevhliu

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -271,7 +271,7 @@ with Diffusers.
 Once an example script works, please make sure to add a comprehensive `README.md` that states how to use the example exactly. This README should include:
 - An example command on how to run the example script as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/dreambooth#running-locally-with-pytorch).
 - A link to some training results (logs, models, ...) that show what the user can expect as shown [here e.g.](https://api.wandb.ai/report/patrickvonplaten/xm6cd5q5).
-- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
+- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
```
Comment (stevhliu) [inline CONTRIBUTING.md:274]:
```suggestion
- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
```

File: PHILOSOPHY.md [hunk 15]
```diff
@@ -31,13 +31,13 @@ As PyTorch states, **explicit is better than implicit** and **simple is better t
...
+is very simple thanks to Diffusers' ability to separate single components of the diffusion pipeline.
 
 ## Tweakable, contributor-friendly over abstraction
 
 For large parts of the library, Diffusers adopts an important design principle of the [Transformers library](https://github.com/huggingface/transformers), which is to prefer copy-pasted code over hasty abstractions. This design principle is very opinionated and stands in stark contrast to popular design principles such as [Don't repeat yourself (DRY)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
-In short, just like Transformers does for modeling files, diffusers prefers to keep an extremely low level of abstraction and very self-contained code for pipelines and schedulers.
+In short, just like Transformers does for modelling files, Diffusers prefers to keep an extremely low level of abstraction and very self-contained code for pipelines and schedulers.
```
Comment (stevhliu) [inline PHILOSOPHY.md:40]: Let's keep the single "L" for model

```suggestion
In short, just like Transformers does for modeling files, Diffusers prefers to keep an extremely low level of abstraction and very self-contained code for pipelines and schedulers.
```

File: PHILOSOPHY.md [hunk 16]
```diff
@@ -82,15 +82,15 @@ Models are designed as configurable toolboxes that are natural extensions of [Py
...
-- The model design is a difficult trade-off between keeping code readable and concise and supporting many model checkpoints. For most parts of the modeling code, classes shall be adapted for new model checkpoints, while there are some exceptions where it is preferred to add new classes to make sure the code is kept concise and
-readable longterm, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
+- The model design is a difficult trade-off between keeping code readable and concise and supporting many model checkpoints. For most parts of the modelling code, classes shall be adapted for new model checkpoints, while there are some exceptions where it is preferred to add new classes to make sure the code is kept concise and
```
Comment (stevhliu) [inline PHILOSOPHY.md:92]:
```suggestion
- The model design is a difficult trade-off between keeping code readable and concise and supporting many model checkpoints. For most parts of the modeling code, classes shall be adapted for new model checkpoints, while there are some exceptions where it is preferred to add new classes to make sure the code is kept concise and
```

File: setup.py [hunk 17]
```diff
@@ -268,30 +270,12 @@ def run(self):
...
+        "Programming Language :: Python :: 3",
+    ]
+    + [
+        f"Programming Language :: Python :: 3.{i}"
+        for i in range(8, version_range_max)
     ],
     cmdclass={"deps_table_update": DepsTableUpdateCommand},
 )
-
-# Release checklist
```
Comment (stevhliu) [inline setup.py:279]: Lets keep the release checklist

**Review by stevhliu — COMMENTED** [review]
Just a few more changes and we should be good to merge :)

**Comment by stevhliu** [comment]
Feel free to ping me for another review after you've addressed all the feedback @standardAI! 🤗

**Comment by standardAI** [comment]
@stevhliu
I should have reviewed the "Pending" comments :/ I didn't know that.

File: README.md (hunk 9 above)
Comment (stevhliu) [reply README.md:2]: I think this should just be when the doc was authored, which in this case, is when Diffusers was created

File: setup.py (hunk 17 above)
Comment (stevhliu) [reply setup.py:279]: The bottom one looks like an abbreviated version. @sayakpaul, is it ok to remove this checklist or would you rather keep it?

File: CITATION.cff (hunk 4 above)
Comment (stevhliu) [reply CITATION.cff:40]: Hmm I guess it's not really a big deal if we remove the version then, since it hasn't been updated in a while. WDYT @sayakpaul?

Comment (standardAI) [reply CITATION.cff:40]: @sayakpaul, WDYT?

File: setup.py (hunk 17 above)
Comment (standardAI) [reply setup.py:279]: @sayakpaul, WDYT?

Comment (standardAI) [reply setup.py:279]: OK, reversed.

File: CITATION.cff (hunk 4 above)
Comment (stevhliu) [reply CITATION.cff:40]: Feel free to just leave it as is and we can update in the future if we need to :)

Comment (standardAI) [reply CITATION.cff:40]: Ok, reversed.

**Review by stevhliu — APPROVED** [review]
Thanks for iterating and improving this PR!

**Comment by standardAI** [comment]
> Thanks for iterating and improving this PR!

Thanks for your feedback!
